src/data.js

```diff
@@ -0,0 +1,23 @@
+let Entity = require('azure-entities');
+
+module.exports = {};
+
+/**
+ * Entity for tracking which task-groups are associated
+ * with which org/repo/sha, etc.
+ *
+ */
+module.exports.Build = Entity.configure({
+  version: 1,
+  partitionKey: Entity.keys.StringKey('taskGroupId'),
+  rowKey: Entity.keys.ConstantKey('taskGroupId'),
+  properties: {
+    organization: Entity.types.String,
```

Comment:
@jonasfj should we rename this to `owner` since it can really be a github user _or_ organization?